tsconfig.json

```diff
@@ -40,7 +40,6 @@
     { "path": "hrm-domain/lambdas/search-index-consumer" },
     { "path": "hrm-domain/lambdas/export-json-writer" },
     { "path": "hrm-domain/hrm-core" },
-    { "path": "hrm-domain/scheduled-tasks/hrm-data-pull" },
```

Comment:
Just to confirm, has the infra that executes this been removed?

Reply:
> Just to confirm, has the infra that executes this been removed?

https://github.com/techmatters/infrastructure-config/pull/488 - it was only active in dev anyway, with this applied it will be gone